internal/services/arckubernetes/arc_kubernetes_flux_configuration_resource_test.go

```diff
@@ -0,0 +1,710 @@
+package arckubernetes_test
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/kubernetesconfiguration/2022-11-01/fluxconfiguration"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type ArcKubernetesFluxConfigurationResource struct{}
+
+func TestAccArcKubernetesFluxConfiguration_basic(t *testing.T) {
+	// generateKey() is a time-consuming operation, we only run this test if an env var is set.
```

Comment:
how long does this usually take?

Reply:
Nearly 2 seconds in my local device. It will cause unit test timeout on GitHub.

Reply:
How will these cause a unit test timeout on GitHub? The acceptance tests aren't run on GH.

Reply:
[The logic](https://github.com/hashicorp/terraform-provider-azurerm/blob/0e9a067248555a143cb3459ae1cba2d26fa00bd5/vendor/github.com/hashicorp/terraform-plugin-testing/helper/resource/testing.go#L737) of not running acc test in unit test is run in `data.ResourceTest`, which is after the code of `generateKey`. So the `generateKey` will still be invoked in unit test.